### ISSUE TITLE
NullPointerException for scheduled execution

### ISSUE THREAD

**Issue by ace-han** [jenkinsci/coordinator-plugin#5]
When you configure scheduled execution, on build perform method you take a NullPointerException.
The cause was on PerformExecutor#prepareExecutionPlan trying to call "parameter.getValue();", but there wasn't a value for CoordinatorParameterValue.PARAM_KEY.
To fix this problem, I implemented the defaultExecutionPlan, then everytime that someone schedule a execution of a coordinator job, it use the default check.

stack trace:
```sh
Started by timer
Building in workspace /var/lib/jenkins/workspace/coordinator
FATAL: null
java.lang.NullPointerException
    at org.jenkinsci.plugins.coordinator.model.PerformExecutor.prepareExecutionPlan(PerformExecutor.java:118)
    at org.jenkinsci.plugins.coordinator.model.PerformExecutor.execute(PerformExecutor.java:88)
    at org.jenkinsci.plugins.coordinator.model.CoordinatorBuilder.perform(CoordinatorBuilder.java:57)
    at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
    at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:804)
    at hudson.model.Build$BuildExecution.build(Build.java:199)
    at hudson.model.Build$BuildExecution.doRun(Build.java:160)
    at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:585)
    at hudson.model.Run.execute(Run.java:1676)
    at hudson.model.Build.run(Build.java:113)
    at hudson.model.ResourceController.execute(ResourceController.java:88)
    at hudson.model.Executor.run(Executor.java:231)

**Comment by ace-han** [comment]
@rodrigozrusso I've taken a close look that the relationship for `projec-builder-build` in this plugin is `1-1-n`, which means that the singleton you mentioned in #3 is accomplished by the `builder.getExecutionPlan().clone(true)` in `CoordinatorProject#newBuild()`. 

Every `build` got its own "namespace" with a unique `orginalExecutionPlan` that won't pollute others'.

So what we to do is just fixing this NPE with an `if-else` instead of introducing a variable like `defaultExecutionPlan`

**Comment by ace-han** [comment]
@rodrigozrusso I've found out how it triggers a scheduled job. In `AbstractProject#scheduleBuild`.

I believed this is the place to avoid this NPE, don't you think?

**Comment by rodrigozrusso** [comment]
@ace-han yes! nice solution.

**Comment by darinpantley** [comment]
I am hitting this bug. When will the fix be released in a new version?

**Comment by ace-han** [comment]
it should be fixed in version 1.1.1

otherwise plz provide some log for me to pinpoint the problem

**Comment by darinpantley** [comment]
Thanks for the quick response! I filed a new bug:
https://github.com/jenkinsci/coordinator-plugin/issues/31
